### ISSUE TITLE
Error with SendString(err) in gorm recipe

### ISSUE THREAD

**Issue by nordbit** [gofiber/recipes#97]
recipes/gorm/book/book.go
Error with c.Status(503).SendString(err)

is: 
```go
func NewBook(c *fiber.Ctx) error {
	db := database.DBConn
	book :=new(Book)
	if err := c.BodyParser(book); err != nil {
		return c.Status(503).SendString(err)
	}
	db.Create(&book)
	return c.JSON(book)
}
```
should be:
```go
func NewBook(c *fiber.Ctx) error {
	db := database.DBConn
	book :=new(Book)
	if err := c.BodyParser(book); err != nil {
		return c.Status(503).SendString(err.Error())
	}
	db.Create(&book)
	return c.JSON(book)
}
```

**Comment by koddr** [comment]
@nordbit hi!

Nice catch! Can you send PR with this fix, please?

**Comment by koddr** [comment]
Merged #98 
Thanks for PR! 👍
